superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -671,180 +607,172 @@ export default class ResultSet extends React.PureComponent<
         )}
       </ReturnedRows>
     );
+  };
+
+  let sql;
```

Comment:
hmm i would have to think about this more, but I wonder if this being in a function is better practice. Either you can put it in the return. Which feels clunky, or maybe create another function for it. I'd have to think through what the differences are.

Reply:
@AAfghahi Let's refactor it in next PR